pkg/operator/operator_test.go

```diff
@@ -512,3 +514,31 @@ func TestMAOConfigFromInfrastructure(t *testing.T) {
 		})
 	}
 }
+
+// TestHandleImagesChange tests that images.json is watched an triggers enqueue
+func TestHandleImagesChange(t *testing.T) {
+	g := NewWithT(t)
+	key := "test"
+	file, err := ioutil.TempFile("", "images.json")
+	defer os.Remove(file.Name())
+	g.Expect(err).ToNot(HaveOccurred())
+
+	stopCh := make(chan struct{})
+	defer close(stopCh)
+	optr := newFakeOperator(nil, nil, stopCh)
+	optr.imagesWatcher, err = fsnotify.NewWatcher()
+	g.Expect(err).To(Succeed())
+	g.Expect(optr.imagesWatcher.Add(file.Name())).To(Succeed())
+
+	go optr.handleImagesChange(key)
```

Comment:
This will only process once, this is non blocking, so why do we put this in a goroutine? Perhaps you meant `go optr.fsWorker()`?